tls/generate-certs.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+
+DOMAIN=pager.localhost
+
+mkdir -p certs
+
+# Create CA if it hasn't already been created
+if [ ! -f certs/ca.crt ];
+then
+echo "Generating a CA to trust locally..."
+openssl genrsa -passout pass:x -out certs/ca.key 2048
+openssl req \
+    -new \
+    -x509 \
+    -days 365 \
+    -sha256 \
+    -key certs/ca.key \
+    -out certs/ca.crt \
+    -config ca.conf
+echo "Reqesting your password to add the generated CA as trusted"
+sudo security add-trusted-cert -d -r trustRoot -k "/Library/Keychains/System.keychain" certs/ca.crt
```

Comment:
this will break for those of us not running OS X

Reply:
I will wrap this in an if, wasn't aware that people were using non-standard machines.